server/command_create.go

```diff
@@ -170,24 +170,24 @@ func (p *Plugin) runCreateCommand(args []string, extra *model.CommandArgs) (*mod
 		license = config.E20License
 	}
 
-	if install.Version != "" {
+	if install.Version != "" || install.Image != "" {
 		err = validVersionOption(install.Version)
 		if err != nil {
 			return nil, true, err
 		}
 
 		var exists bool
-		repository := "mattermost/mattermost-enterprise-edition"
-		exists, err = p.dockerClient.ValidTag(install.Version, repository)
+
+		exists, err = p.dockerClient.ValidTag(install.Version, install.Image)
```

Comment:
This change means that the image must always be specified, now. If only the version is specified, I think it would be better to assume that the user does not wish to change the image and reuse whatever the current image is instead of returning an error. I'm 1/5 on this, what do you think @gabrieljackson ?

Reply:
You are correct that `mattermost-enterprise-edition` should be our default repo, and it still is via the default value of the image flag in the create command. I think this is good to keep as is.